app/assets/stylesheets/govuk_publishing_components/components/govspeak/_content-block.scss

```diff
@@ -0,0 +1,32 @@
+@import "govuk/core/links";
+@import "govuk/core/lists";
+@import "govuk/core/typography";
```

Comment:
I think you can replace all these specific lines with 

```
@import "govuk_publishing_components/individual_component_support";
```

...which ultimately imports `govuk/base`, which I think includes all of this stuff? Worth trying out.